src/components/Homepage/Icon.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import { Link } from "@reach/router";
+import styled from "@emotion/styled";
+
+interface IProps {
+  src: string;
+  url: string;
+  title: string;
+  to: string;
+}
+const Oval: any = styled.span`
+  background-image: url(${(props: any) => props.url});
```

Comment:
I don't think this line is ok, It's working but there is any and also ${}

Reply:
I can't find any other solution for this.

Reply:
fix for this

```
interface IOval {
  url: string;
}

const Oval = styled.div<IOval>`
  background: url(${props => props.url}) no-repeat center;
`;
```